### DISCUSSION TITLE
CoursePlay editor

### DISCUSSION THREAD

**Discussion by prawler74** [Courseplay/Courseplay_FS22#1790] [Q&A] [answered]
Hi. I really like the new added festure. But here is the thing. I see in the menu that i am supposef to "move" the points. I press the right mousebutton. But i am not able to edit/move anything. What am i missing?

Bjørn

**Comment by Tensuko** [comment, marked as answer]
Did you select the "move" option ?
Then use the mouse of get on the waypoint (it lights up a bit when you drag it) then move it arround and drop it again.

**Reply by prawler74** [reply]
Hi. 

I tried eveery option available...i did se a grey circle on the ground. Tried to align that to the waypoint. 


I am supposed to click the yellow "arrow"?

I will try again...i believe i have tried everything....will test and update. 

Best regards
Bjørn 😊

**Reply by prawler74** [reply]
I dont know what happened. Bit noe it seems to work. 

Yet another thing to learn. 

Thanx. Keep up the good work. Awesome piece of "software"

Bjørn
